_commands/basics/echo.md

```diff
@@ -0,0 +1,68 @@
+---
+---
+
+echo
+--
+
+`echo` is used to output whatever it is given into stdio. Often useful see the value of enviromental variables.
```

Comment:
nit: it echoes to "stdout", not stdio (that's a library used by C/C++ to read/write stdin, stdout, and stderr)